keylime/tenant.py

```diff
@@ -467,11 +489,13 @@ def validate_tpm_quote(self, public_key: str, quote: str, hash_alg: algorithms.H
             [type] -- [description]
         """
         if self.registrar_data is None:
-            logger.warning("AIK not found in registrar, quote not validated")
+            logger.warning(
+                "AIK not found in registrar %s, quote not validated for %s", self.registrar_fid_str, self.agent_fid_str
```

Comment:
this will print "... in registrar Registrar ..."

Reply:
indeed, fixed